core/designsystem/src/main/java/org/mifos/mobilewallet/mifospay/designsystem/component/ProfileConcentricImage.kt

```diff
@@ -0,0 +1,44 @@
+package org.mifos.mobilewallet.mifospay.designsystem.component
+
+import android.graphics.Bitmap
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun ProfileImage(bitmap: Bitmap?) {
```

Comment:
Same,

`This compose screen should be part of the profile package not designsystem, designsystem is only core building block of the app.`